samples/metrics.js

```diff
@@ -30,10 +30,7 @@ async function createMetricDescriptor(projectId) {
   // Creates a client
   const client = new monitoring.MetricServiceClient();
 
-  /**
```

Comment:
Greetings!  These are here on purpose :) We don't want to assume developers running the same have the `GOOGLE_CLOUD_PROJECT` environment variable set on their current process.  We cover a lot of the motivation for this here:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

Reply:
What confuses me is that this is inside a function where projectId is optional, so when I uncomment that line I get

`SyntaxError: Identifier 'projectId' has already been declared`

How should I fix this line while still avoiding the assumption of the presence of environment variables?

Reply:
For the code you have now, that makes sense.  You want to leave those sections exactly as they were :) The projectId is passed into the `main` function from the outside.  You should be able to return all the projectId code that was modified back to it's original form.

Reply:
I've reverted the `projectId` code back to its original form. From this state, I then uncomment this line so it goes from
> // const projectId = 'YOUR_PROJECT_ID';
to
> const projectId = 'YOUR_PROJECT_ID';

Then I run
> node samples/metrics.js create
and I get
> SyntaxError: Identifier 'projectId' has already been declared